### DISCUSSION TITLE
Drop-down locale selector

### DISCUSSION THREAD

**Discussion by amrita-sakhrani** [twilio-labs/paste#2666] [Office hours] [answered]
Hi, as part of the localisation effort we are working to launch in Portuguese in Q4. We will need to design and create a locale selector for the user 
• Selector to be used on sign-up, onboarding, Ahoy and inside the console user settings 
• Some other platforms have a beta label next to the locale that is still in the works 
• What would be the best component to achieve this? 

Stage: Early exploration 
Figma: https://www.figma.com/file/SlOYUexspe2eQG5nZRjxNn/Locale-Selection?node-id=956%3A40937

**Comment by nkrantz** [comment, marked as answer]
Hi @amrita-sakhrani! Thanks for coming to Office Hours today. [Here's the link to the recording](https://www.loom.com/share/3305984498bf40179a4490579a0f2203) (check the #help-design-system channel for the password).

To recap, you can continue using [the Combobox component](https://paste.twilio.design/components/combobox) for the selector like you currently have in your design explorations. In order to compose the options in Figma, you can either detach the component or create a local component. Let us know if you have any more questions about how to do that.

I'd recommend touching base with other folks working on language/locale selectors. You can follow #2580 for updates on the new icon we talked about today.

Please keep us updated on your progress and let us know if you have any questions! Feel free to keep using this thread for any updates.
